smart-mon.spec

```diff
@@ -5,6 +5,8 @@
 %define bin_dir /usr/bin
```

Comment:
This is actually already defined, can remove this completely and just use `%{_bindir}`.